src/streaming/StreamProcessor.js

```diff
@@ -184,20 +234,74 @@ function StreamProcessor(config) {
         return representationController ? representationController.isUpdating() : false;
     }
 
+    function onStreamInitialized(e) {
+        if (!e.streamInfo || streamInfo.id !== e.streamInfo.id) return;
+
+        if (!streamInitialized) {
+            streamInitialized = true;
+            if (isDynamic) {
+                timelineConverter.setTimeSyncCompleted(true);
+                setLiveEdgeSeekTarget();
+            } else {
+                const seekTarget = playbackController.getStreamStartTime(false);
+                bufferController.setSeekStartTime(seekTarget);
+                scheduleController.setCurrentRepresentation(getRepresentationInfo());
```

Comment:
Before this was also set for livestreams. Might need to move this up to also be applied to livestreams

Reply:
This is done in setLiveEdgeSeekTarget() method.
This is the way it was done before in ScheduleController